### PR TITLE
Feature#26/crud employee

### LINKED ISSUE #26: Crud for employees
# REQUIREMENTS
Implement the functional crud on the employees view for the adminsitrator
- Use the view already made
- On deactivate update position to "Disabled" 
- On update send to the update view
- On add redirect to new view

### PR DESCRIPTION
**Pull request by nallely64**
# Description
 - What?: We've added the employee CRUD.
 - Why?: To allow the admin to have the full control of the employees.
 For more information see issue #26 

  ## Screenshots
image (4)

### CODE REVIEW

File: app/assets/stylesheets/utils/_variables.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-$white_button: #FFFFFF;
-$white_title: #FFFFFF;
+$activate_button: #01b50c;
+$edit_button: #f1d117;
+$red_button_hover: #f65058aa;
+$red_button: #f65058ff;
+$wheat_button: #f7e9c6;
+$wheat_color: #f7e9c6;
+$white_button: #ffffff;
+$white_title: #ffffff;
```
Comment (YilKanda) [inline app/assets/stylesheets/utils/_variables.scss:15]: Do you really occupy all these colors?

File: config/locales/simple_form.en.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    # hints:
+    #   defaults:
+    #     username: 'User name to sign in.'
+    #     password: 'No special characters, please.'
+    # include_blanks:
+    #   defaults:
+    #     age: 'Rather not say'
+    # prompts:
+    #   defaults:
+    #     age: 'Select your age'
```
Comment (YilKanda) [inline config/locales/simple_form.en.yml:31]: Code that you do not use, delete it.

File: app/assets/stylesheets/utils/_variables.scss (hunk 1 above)
Comment (AlejandroCuriel) [reply app/assets/stylesheets/utils/_variables.scss:15]: yes, each color is being used

File: config/locales/simple_form.en.yml (hunk 2 above)
Comment (juperameza) [reply config/locales/simple_form.en.yml:31]: Done

**Review by VictorElizalde — APPROVED** [review]
LGTM, loved simple-form

**Review by Javi1499 — APPROVED** [review]
LGTM!!!!!!!!!!!!!!

**Review by femm34 — APPROVED** [review]
lgtm

**Review by EliasUR — APPROVED** [review]
gg

**Pull request merged**